backend/connector-integration/src/connectors/razorpay/transformers.rs

```diff
@@ -433,9 +451,9 @@ pub struct RazorpayPsyncResponse {
     pub id: String,
     pub entity: String,
     pub amount: i64,
-    pub base_amount: i64,
+    pub base_amount: Option<i64>,
     pub currency: String,
-    pub base_currency: String,
+    pub base_currency: Option<String>,
```

Comment:
```suggestion
    pub base_currency: Option<enums::Currency>,
```

Make this change at all the places